apps/alb_monitor/mix.exs

```diff
@@ -0,0 +1,60 @@
+defmodule ALBMonitor.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :alb_monitor,
+     aliases: aliases(),
+     build_embedded: Mix.env == :prod,
+     build_path: "../../_build",
+     config_path: "../../config/config.exs",
+     deps: deps(),
+     deps_path: "../../deps",
+     elixir: "~> 1.3",
+     elixirc_paths: elixirc_paths(Mix.env),
+     lockfile: "../../mix.lock",
+     start_permanent: Mix.env == :prod,
+     test_coverage: [tool: ExCoveralls],
+     version: "0.1.0"]
+  end
+
+  # Configuration for the OTP application
+  #
+  # Type "mix help compile.app" for more information
+  def application do
+    [extra_applications: [:logger],
+     mod: {ALBMonitor, []}]
+  end
+
+  defp aliases do
+    [compile: ["compile --warnings-as-errors"]]
+  end
+
+  # Dependencies can be Hex packages:
+  #
+  #   {:mydep, "~> 0.3.0"}
+  #
+  # Or git/path repositories:
+  #
+  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
+  #
+  # To depend on another app inside the umbrella:
+  #
+  #   {:myapp, in_umbrella: true}
+  #
+  # Type "mix help deps" for more examples and options
+  defp deps do
+    [
+      {:ex_aws, "~> 2.0"},
+      {:ex_aws_elastic_load_balancing, "~> 2.0"},
+      {:httpoison, "~> 1.7"},
+      {:mox, "~> 1.0", only: :test},
+      {:poison, "~> 3.1"},
```

Comment:
I noticed we already had Poison installed from elsewhere in the umbrella and wanted to avoid using two different JSON libraries, but on closer inspection we are _also_ already using Jason. Might as well switch this, then.